### PR TITLE
Add davecheney/httpstat

### PR DESCRIPTION
**Pull request by piksu**
Hey,

First contribution. I felt like this tool fits the criteria for Main bucket.

See: https://github.com/davecheney/httpstat

### CODE REVIEW

**Comment by linsui** [comment]
Please add `checkver` and `autoupdate` .

File: bucket/httpstat.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        "32bit": {
+            "url": "https://github.com/davecheney/httpstat/releases/download/v1.0.0/httpstat-windows-386-v1.0.0.exe#/httpstat.exe",
+            "hash": "9B43886614830830A84FAE87CBBB2F8EF01B09B1986A3A4F4AA77978887BAEE3"
+        },
+        "64bit": {
+            "url": "https://github.com/davecheney/httpstat/releases/download/v1.0.0/httpstat-windows-amd64-v1.0.0.exe#/httpstat.exe",
+            "hash": "7CF510C8927A23DB19E0D579BF44C3C325ABE0760E34FEA20567461759B794E9"
+        }
+    },
+    "checkver": {
```
Comment (linsui) [inline bucket/httpstat.json]:
```suggestion
    "checkver": "github",
```

File: bucket/httpstat.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "32bit": {
+                "url": "https://github.com/davecheney/httpstat/releases/download/v$version/httpstat-windows-386-v$version.exe#/httpstat.exe"
+            },
+            "64bit": {
+                "url": "https://github.com/davecheney/httpstat/releases/download/v$version/httpstat-windows-amd64-v$version.exe#/httpstat.exe"
+            }
+        }
+    },
+    "bin": "httpstat.exe"
+}
```
Comment (linsui) [inline bucket/httpstat.json]:
```suggestion
}

```

File: bucket/httpstat.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        "architecture": {
+            "32bit": {
+                "url": "https://github.com/davecheney/httpstat/releases/download/v$version/httpstat-windows-386-v$version.exe#/httpstat.exe"
+            },
+            "64bit": {
+                "url": "https://github.com/davecheney/httpstat/releases/download/v$version/httpstat-windows-amd64-v$version.exe#/httpstat.exe"
+            }
+        }
+    },
+    "bin": "httpstat.exe"
```
Comment (linsui) [inline bucket/httpstat.json]: Please put `bin` before `checkver`.

File: bucket/httpstat.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/davecheney/httpstat",
+    "description": "httpstat visualizes curl(1) statistics in a way of beauty and clarity.",
```
Comment (Ash258) [inline bucket/httpstat.json]:
```suggestion
    "description": "curl(1) statistics visualizer.",
```

File: bucket/httpstat.json (hunk 1 above)
Comment (piksu) [reply bucket/httpstat.json]: By defining checkver to "github", the checkver.ps1 is able to figure out how to fetch the latest version by looking homepage/architecture urls?

File: bucket/httpstat.json (hunk 2 above)
Comment (piksu) [reply bucket/httpstat.json]: Fixed.

File: bucket/httpstat.json (hunk 3 above)
Comment (piksu) [reply bucket/httpstat.json]: Fixed.

File: bucket/httpstat.json (hunk 1 above)
Comment (piksu) [reply bucket/httpstat.json]: Applied.

File: bucket/httpstat.json (hunk 4 above)
Comment (piksu) [reply bucket/httpstat.json]: This have been applied.

File: bucket/httpstat.json (hunk 1 above)
Comment (linsui) [reply bucket/httpstat.json]: Yeah, see https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#special-cases

**Pull request merged**